### PR TITLE
Website update prep

### PR DESCRIPTION
**Pull request by lukewilliamboswell**
@rtfeldman discussed changes for the 2023 website update in [this tulip thread](https://roc.zulipchat.com/#narrow/stream/316715-contributing/topic/2023.20website.20update/near/361800073). 

This PR is an attempt at putting together a structure at a high level of abstraction for the website content.

### CODE REVIEW

**Comment by lukewilliamboswell** [comment]
Some progress on the home page

Screenshot 2023-06-25 at 14 22 27

@Anton-4 - how do you feel about setting up a subdomain where we could host this WIP site while it is being developed?

**Comment by Anton-4** [comment]
I've checked it out and it doesn't look too difficult, I'll get on it tomorrow.

**Comment by Anton-4** [comment]
Didn't get to it yet, hopefully tomorrow.

**Comment by Anton-4** [comment]
Now that I think of it, it would be easier to just create a new wipwebsite folder in `www/build` or is there a reason that a subdomain would be better?

**Comment by rtfeldman** [comment]
I'm fine with either approach 👍

File: www/build.sh [hunk 1]
```diff
@@ -81,17 +82,22 @@ else
...
-  rm -rf roc_nightly roc_releases.json
 fi
 
+$roc version
+$roc run www/generate_tutorial/src/tutorial.roc -- www/generate_tutorial/src/input/ www/build/tutorial/
 mv www/build/tutorial/tutorial.html www/build/tutorial/index.html
 
+# for new wip site
+mkdir www/build/wip
+roc run www/wip_new_website/main.roc -- www/wip_new_website/content/ www/build/wip
```
Comment (Anton-4) [inline www/build.sh:97]: @lukewilliamboswell, I moved the build steps to this script, I really wanted to use arguments with the roc build script but couldn't due to https://github.com/roc-lang/basic-cli/issues/82

**Comment by Anton-4** [comment]
Is this good to merge @lukewilliamboswell?

**Review by Anton-4 — APPROVED** [review]

**Pull request merged**